tokens.xml

```diff
@@ -7098,6 +7247,21 @@ This creature can't be enchanted.</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>The Atropal</name>
+            <text>Deathtouch</text>
+            <prop>
+                <colors>B</colors>
+                <type>Token Legendary Creature — God</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>4/4</pt>
+            </prop>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/6/5/65f8e40f-fb5e-4ab8-add3-a8b87e7bcdd9.jpg?1626139916">AFR</set>
+            <!-- this token is created by another token only -->
```

Comment:
Why not reverse-link to the `Tomb of Annihilation` dungeon token here?

Instead you linked it the other way around :) https://github.com/Cockatrice/Magic-Token/pull/108/files#diff-ef2c5b2e0216513ba7314de2c6637a3b48b09a5ea3ee44d13c951195fb293697R8783

Reply:
I did it as related because there is precedence for it in other tokens such as the Ajani emblem that is related to cat tokens. There are a few others as well. It doesn't make a difference either way I suppose but we should keep it consistent one way or the other.